File: weibo/weibo/Classes/View/Home/Model/WBStatus.m

//
//  WBStatus.m
//  weibo
//
//  Created by 叮咚钱包富银 on 2017/11/6.
//  Copyright © 2017年 叮咚钱包富银. All rights reserved.
//

#import "WBStatus.h"

@implementation WBStatus

+ (NSDictionary *)modelContainerPropertyGenericClass {
    return @{
             @"pic_urls" : [WBStatusPictureModel class]
             
             };
}

- (NSString *)description {
    return self.yy_modelDescription;
}
@end

File: weibo/weibo/Classes/Model/WBStatus.swift


//
//  WBStatus.swift
//  weibo
//
//  Created by 叮咚钱包富银 on 2017/10/27.
//  Copyright © 2017年 叮咚钱包富银. All rights reserved.
//

import UIKit
import YYModel

class WBStatus: NSObject {

    var id:Int64 = 0
    
    var text: String?
    
    override var description: String {
        return yy_modelDescription()
    }
}
